components/Profile/VotingPosition.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const VotingPosition = ({data}) => {
+    if(data.amendment.length > 0){
```

Comment:
you should also check for ``` data.bill.bill_id``` there are voting positions showing up with any id or information